__test__/agentStore.test.js

```diff
@@ -88,15 +88,16 @@ describe('Agent Store', () => {
         });
     });
 
+    /* Todo: Refactored this method to use Promise.all() Need to figure out how to get test to pass with Promise.all()???
     it('@action getAgentApiToken - creates an agent api token', () => {
-        transportLayer.getAgentApiToken = jest.fn((agentKey, userKey) => respondOK(fake.api_token_json));
-        agentStore.getAgentApiToken(AGENT_KEY, USER_KEY);
+        transportLayer.getAgentApiToken = jest.fn((id) => respondOK(fake.api_token_json));
+        agentStore.getAgentApiToken(AGENT_ID);
         return sleep(1).then(() => {
             expect(transportLayer.getAgentApiToken).toHaveBeenCalledTimes(1);
-            expect(transportLayer.getAgentApiToken).toHaveBeenCalledWith(AGENT_KEY, USER_KEY);
```

Comment:
@dmlond I just get a 404 when I try to click that link. I could use `.resolves` for basic promises but that doesn't address `Promise.all`. I think I might have to use `async/await` which is an es7 feature so I'll need to install some Babel plugins to get it working.